versions.txt

```diff
@@ -2,7 +2,7 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest 
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
```

Comment:
nit: which docs is this referring to? if not the changelog, then are they being updated independently?

Reply:
I think this might be a copy/paste issue from Jaeger's, but it's referring to the readme/examples.